doc/authoring_command_modules/authoring_commands.md

```diff
@@ -34,11 +49,13 @@ from azure.cli.commands import cli_command
 
 The signature of this method is 
 ```Python
-def cli_command(name, operation, client_factory=None, transform=None, table_transformer=None):
+def cli_command(module_name, name, operation, client_factory=None, transform=None, table_transformer=None):
 ```
 You will generally only specify `name`, `operation` and possibly `table_transformer`.
+  - `module_name` - The name of the module that is registering the command (e.g. `azure.cli.command_modules.vm.commands`). Typically this will be `__name__`.
   - `name` - String uniquely naming your command and placing it within the command hierachy. It will be the string that you would type at the command line, omitting `az` (ex: access your command at `az mypackage mycommand` using a name of `mypackage mycommand`).
-  - `operation` - Your function's name.
+  - `operation` - The handler that will be executed. Format is `<module_to_import>#<attribute_list>`
```

Comment:
Does this work the same with custom commands? I imagine many people might pass the method "the old way" so do we check that operation is a string and throw a useful error message if they pass a function?